src/Uno.UWP/Globalization/ApplicationLanguages.cs

```diff
@@ -1,15 +1,16 @@
 ﻿using System;
 using System.Globalization;
 using System.Linq;
-using System.Reflection;
 using System.Text.RegularExpressions;
 using Uno.UI;
+using Windows.Storage;
 
 namespace Windows.Globalization
 {
 	public static partial class ApplicationLanguages
 	{
-		private static string _primaryLanguageOverride;
+		private static string _primaryLanguageOverride = string.Empty;
+		private const string PrimaryLanguageOverrideSettingKey = "Uno.PrimaryLanguageOverride";
```

Comment:
We're using `__` as a prefix for other related local settings added by uno:

```suggestion
		private const string PrimaryLanguageOverrideSettingKey = "__Uno.PrimaryLanguageOverride";
```